plugins/org.eclipse.papyrus.uml.interaction.model/META-INF/MANIFEST.MF

```diff
@@ -24,9 +24,11 @@ Export-Package: org.eclipse.papyrus.uml.interaction.internal.model;
  org.eclipse.papyrus.uml.interaction.internal.model.commands;x-internal:=true,
  org.eclipse.papyrus.uml.interaction.internal.model.impl;
   x-friends:="org.eclipse.papyrus.uml.diagram.sequence.runtime",
- org.eclipse.papyrus.uml.interaction.internal.model.spi.impl;x-internal:=true,
+ org.eclipse.papyrus.uml.interaction.internal.model.spi.impl;
+  x-friends:="org.eclipse.papyrus.uml.interaction.model.tests",
```

Comment:
Rather than reference a test bundle from a deployed production bundle, we can instead configure access rules in the `.classpath` file of the tests project to make the internals of this bundle accessible to it without warnings.